### DISCUSSION TITLE
Error building docker - nvblox_torch installation

### DISCUSSION THREAD

**Discussion by MarcoPereira00** [NVlabs/curobo#271] [Software Q&A]
1. cuRobo installation mode : docker isaac sim
2. python version: 3
3. Isaac Sim version (if using): 2022.2.1

When I try to install nvblox_torch with `sh install_isaac_sim.sh $($omni_python -c 'import torch.utils; print(torch.utils.cmake_prefix_path)') && \
    $omni_python -m pip install -e .` I get this error:

![image](https://github.com/NVlabs/curobo/assets/75946345/d7f50134-f8d8-4e51-836b-410bd69b308f)

**Comment by balakumar-s** [comment]
What version of nvidia driver are you using?

And are you using this dockerfile: https://github.com/NVlabs/curobo/blob/main/docker/isaac_sim.dockerfile

**Comment by MarcoPereira00** [comment]
Im using 535 and yes Im using that dockerfile

**Comment by balakumar-s** [comment]
Did you add this to your docker json file? 
https://curobo.org/get_started/5_docker_development.html#building-your-own-docker-image-with-curobo
